src/client/routes/nfc/Nfc.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { FunctionComponent, useState } from 'react';
+import Select from 'react-select';
+import styled from 'styled-components';
+import { SearchBox } from '../../components/Input/SearchBox';
+import { ToggleSwitch } from '../../components/Buttons/ToggleSwitch';
+import { ActionButton } from '../../components/Buttons/ActionButton';
+
+const EventSelect = styled(Select)``;
+const CHECK_IN_VALUE = 'check_in'; // TODO
+
+// TODO(timliang/kenny): Connect this to events data
+const EventOptions: { label: string; value: string }[] = [
+	{ label: 'Hackathon Check-in', value: CHECK_IN_VALUE },
+	{ label: 'PlaceholderEvent1', value: 'test_1' },
+	{ label: 'PlaceholderEvent2', value: 'test_2' },
+];
+
+export const Nfc: FunctionComponent = (): JSX.Element => {
+	const [manualMode, setManualMode] = useState(false);
+	const [unadmitMode, setUnadmitMode] = useState(false);
+	const [eventSelected, setEventSelected] = useState(CHECK_IN_VALUE);
+	return (
+		<div>
```

Comment:
style: To be consistent with the rest of the pages, this might look good inside the `FloatingPopup` container like profile, application, and hackertable.

Reply:
Done.